pybamm/solvers/c_solvers/idaklu.cpp

```diff
@@ -17,6 +18,7 @@ Function generate_function(const std::string &data)
 namespace py = pybind11;
 
 PYBIND11_MAKE_OPAQUE(std::vector<np_array>);
+//PYBIND11_MAKE_OPAQUE(std::vector<Function>);
```

Comment:
do we need this commented out code anymore?

Reply:
removed